communication/src/main/java/org/jboss/da/communication/aprox/model/VersionResponse.java

```diff
@@ -0,0 +1,59 @@
+package org.jboss.da.communication.aprox.model;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "metadata")
+public class VersionResponse {
+
```

Comment:
Would it be worth considering using @jdcasey's Atlas which already has the concept of a ProjectVersionRef ( https://github.com/Commonjava/atlas/tree/master/identities/src/main/java/org/commonjava/maven/atlas/ident/ref) and I believe support for serializing them ?